.github/workflows/release.yml

```diff
@@ -25,10 +25,19 @@ jobs:
         uses: cachix/install-nix-action@v16
         with:
           nix_path: nixpkgs=channel:nixos-21.11
-      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
Oh, we never needed this, did we? We get it with our nix config.

Reply:
Thats right. I was reviewing our pattern and realized it's unnecessary